denops/@ddc-sources/path.ts

```diff
@@ -110,3 +147,9 @@ export class Source extends BaseSource<Params, UserData> {
     ) as DirSeparator;
   }
 }
+
+export const _internals = {
```

Comment:
I am against using `_`. Consider using `export function ...`

Reply:
They are exporting to test internal functions.
This technique is also used in testing Deno's standard library.